tests/Functional/FunctionalTest.php

```diff
@@ -201,7 +201,8 @@ private function findSubDirectories(string $directory): iterable
 
         return $finder
             ->directories()
-            ->in($directory);
+            ->in($directory)
+            ->notName('_*');
```

Comment:
Let's document this trick in `CONTRIBUTING.md` or wherever you see fit?

Reply:
Until now we have not documented at all how to create Functional tests in our project. Not sure if it needs to be documented either as there are many things that also need to be documented and would help a broader number of developers. 
So if we document it I would do it in the PR that documents how to add tests in General

Reply:
Ok, I'll create an issue to track this.